sys/vane/clay.hoon

```diff
@@ -318,6 +325,7 @@
           {$note p/@tD q/tank}                          ::  debug message
           {$ogre p/@tas}                                ::  delete mount point
           {$writ p/riot}                                ::  response
+          {$wris p/(set rant)}                          ::  responses
       ==                                                ::
```

Comment:
Is there a reason this arm exists separately from zuse? `++move` directly above uses the zuse version instead, so it's not like this copy of the definition is ever referenced. (In particular, the `%wris` definition is outdated and would not otherwise compile)

Reply:
Hmm yeah, I missed this for the reason you mentioned. Originally updates it because "hey, this is here, might as well keep it consistent", but you're right that there's no reason for it to be there in the first place. Can confirm clay compiles just fine without it.

Similarly, replacing `++note`'s body with `note-arvo` (and making sure to say `%b` where we currently say `%t`) works fine too. Probably best to not re-define these interfaces locally, I imagine?